kythe/go/platform/tools/kzip/createcmd/createcmd.go

```diff
@@ -229,6 +231,28 @@ func (cb *compilationBuilder) done() error {
 	return cb.out.Close()
 }
 
+func (cb *compilationBuilder) tryMakeRelative(path string) string {
```

Comment:
Document "relative to `WorkingDirectory` or $PWD" or returns the full path if not relative.